A32NX/html_ui/Pages/A32NX_Core/A32NX_LocalVarUpdater.js

```diff
@@ -2,6 +2,9 @@
 // To create and sync a new local simvar, you need to add a selector and an updater.
 // The selector calculates the new value based on other simvars and some logic.
 // The updater compares the new value from the selector with the current value from the local simvar,
+
+const { formatDiagnosticsWithColorAndContext } = require("typescript");
```

Comment:
Debug code?

Reply:
Not guilty. But could be. I'll remove it.